package.json

```diff
@@ -20,6 +20,7 @@
     "npm": ">=7.0.0"
   },
   "dependencies": {
+<<<<<<< Updated upstream
```

Comment:
Package.json contains stashed changes/merge conflicts

Reply:
@Juilzze resolved this issue in latest commit.